File: snippets/snippets/python/chat-v2-post/documents.py
import cohere

co = cohere.ClientV2()

response = co.chat(
    model="command-a-03-2025",
    documents=[
        {"id": "1", "data": {"text": "Cohere is the best!", "title": "The best"}}
    ],
    messages=[{"role": "user", "content": "Who's the best?"}],
)

print(response)

File: snippets/snippets/node/chat-v2-post/documents.ts
const { CohereClientV2 } = require('cohere-ai');

const cohere = new CohereClientV2({});

(async () => {
  const response = await cohere.chat({
    model: 'command-a-03-2025',
    documents: [{ id: '1', data: 'Cohere is the best!' }],
    messages: [
      {
        role: 'user',
        content: [{ type: 'text', text: "Who's the best?" }],
      },
    ],
  });

  console.log(response);
})();
